castle/cms/browser/controlpanel/status.py

```diff
@@ -0,0 +1,55 @@
+from Products.Five.browser import BrowserView
+import subprocess
+import redis
+import celery
+import celery.bin.base
+import celery.bin.celery
+import celery.platforms
+
+
+class StatusView(BrowserView):
+
+    def docsplit(self):
+        retval = self.DocsplitSearch()
+        if retval[0]:
+            status = 'OK'
+            icon = '<span class="glyphicon glyphicon-ok" aria-hidden="true"></span> Docsplit: '
+            return '<span>' + icon + status + '</span>'
+        if not retval[0]:
+            status = 'ERROR: ' + retval[1]
+            icon = '<span class="glyphicon glyphicon-remove" aria-hidden="true"></span> Docsplit: '
+            return '<span>' + icon + status + '</span>'
+
+    def elasticsearch(self):
+        rs = redis.Redis("localhost")
```

Comment:
Are we certain that redis will always be connecting to localhost? Is there a deployment case for having redis running on a different IP address from the Castle site?